Documentation/Books/Manual/Security/Auditing/AuditEvents.md

```diff
@@ -117,36 +120,36 @@ Documents
 ### Reading a single document
```

Comment:
the headline says **Reading** but the logged message is `create document ...`?

Reply:
Ah, good catch! Should be fixed now.